cratedb_toolkit/docs/settings.py

```
@@ -0,0 +1,773 @@
+@click.command(context_settings={"help_option_names": ["-h", "--help"]})
+@click.option(
+    "--format",
+    "-f",
+    type=click.Choice(["json", "markdown", "sql"]),
+    default="json",
+    help="Output format (json, markdown or sql)",
+)
+@click.option("--output", "-o", default=None, help="Output file name")
+@click.option("--verbose", "-v", is_flag=True, help="Enable verbose output")
+@click.version_option(version="1.0.0")
+def main(format: str, output: str, verbose: bool):
```

Ruff (0.8.2)

Function argument `format` is shadowing a Python builtin

(A002)